improbable-research/keanu
# Add enough python bindings to pass coal mining example

## Steps to run tests for now
1. Ensure you have python 3.6 installed
2. Run `gradle shadowJar` from root folder
3. Run `pipenv install --dev` from keanu-python folder
4. Run `pipenv shell` and run `pytest tests` from keanu-python folder

## Steps to install keanu using pip in your local environment
3. Run `pip install -e .` in keanu-python folder
